target/linux/ramips/image/mt7621.mk

```diff
@@ -286,6 +286,19 @@ define Device/arcadyan_we420223-99
 endef
 TARGET_DEVICES += arcadyan_we420223-99
 
+define Device/zte_mf286c3
+  $(Device/nand)
+  $(Device/uimage-lzma-loader)
+  VID_HDR_OFFSET := 2048
+  IMAGE_SIZE := 41984k
+  KERNEL_SIZE := 5120k
+  DEVICE_VENDOR := ZTE
+  DEVICE_MODEL := MF286C3
+  DEVICE_PACKAGES := kmod-mt7603 kmod-usb2 kmod-usb3 kmod-mt7615e kmod-i2c-core \
```

Comment:
MT7621 doesn't need package `kmod-usb2`. And perhaps you want `kmod-i2c-ralink` instead of the `kmod-i2c-core` here.